source/PluginDev/Assets/GooglePlayGames/Platforms/Android/AndroidClient.cs

```diff
@@ -42,7 +42,7 @@ private enum AuthState
         private readonly object AuthStateLock = new object();
 
         private readonly PlayGamesClientConfiguration mConfiguration;
-        private volatile ITurnBasedMultiplayerClient mTurnBasedClient;
+        private volatile AndroidTurnBasedMultiplayerClient mTurnBasedClient;
```

Comment:
Converted from ITurnBasedMultiplayerClient to AndroidTurnBasedMultiplayerClient to address https://github.com/playgameservices/play-games-plugin-for-unity/pull/2577/files/bd5239864ec45c5855e266415ec22ee8f72cc45e..f1ede0601f94754e3314e8d717368b07431fcdf2#r294235778